src/sentry/static/sentry/app/components/forms/booleanField.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import InputField from './inputField';
+
+export default class BooleanField extends InputField {
+  onChange(e) {
+    this.setState({
+      value: e.target.checked,
+    }, () => {
+      this.props.onChange(this.state.value);
+    });
+  }
+
+  getField() {
+    return (
+      <input id={this.getId()}
+          type={this.getType()}
+          style={{marginLeft: '10px'}}
+          onChange={this.onChange.bind(this)}
+          disabled={this.props.disabled}
+          defaultChecked={this.state.value} />
```

Comment:
It seems this is the correct use: https://facebook.github.io/react/docs/forms.html#default-value